### DISCUSSION TITLE
3DFT and iTwin workflow

### DISCUSSION THREAD

**Discussion by DinosIpiotis** [iTwin/community#16] [Q&A]
Hi there,

I am using the files from GitHub to set up and stream an iTwin model to Unreal Engine. All good so far, I have opened the project file, created an app and I have an existing iModel. I cannot understand how to tie the iModel to my app so that it can be picked up and streamed.

Can someone assist?

**Comment by pratikshan85** [comment]
Please follow the below link :
https://github.com/iTwin/unreal-engine-3dft-plugin

Also, refer to the below image that explains how to select the iModel and its version.

![image](https://github.com/iTwin/community/assets/48123387/3fd1e561-4851-411a-b3b0-08f86750a699)
